package.nls.json

```diff
@@ -52,6 +52,7 @@
     "vscode-docker.tasks.docker-build.dockerBuild.dockerfile": "Path to the Dockerfile used for the build.",
     "vscode-docker.tasks.docker-build.dockerBuild.labels.description": "Labels applied to the Docker image used for debugging.",
     "vscode-docker.tasks.docker-build.dockerBuild.labels.includeDefaults": "Whether to include the default set of labels defined by the Docker extension",
+    "vscode-docker.tasks.docker-build.dockerBuild.platform": "Target platform for the image build",
```

Comment:
Good question. I originally thought to do multiple since that's what I saw in examples, but upon closer inspection, that applies _only_ to the `docker buildx build` command--_not_ the regular `docker build` command, which only accepts a single platform option.

Reply:
Also, in tasks.json it will give a warning squiggle if you try to give an array instead of a single value.

Reply:
![image](https://user-images.githubusercontent.com/36966225/229164985-8c98de9f-e58d-49ae-b8b3-277cc8c6e27a.png)

Reply:
Yup. That's the one.